manager/src/main/java/org/apache/hertzbeat/manager/service/impl/TagServiceImpl.java

```diff
@@ -60,6 +60,10 @@ public class TagServiceImpl implements TagService {
     public void addTags(List<Tag> tags) {
```

Comment:
👍 hi, how about the existed same name and value when modifyTag

Reply:
okay, done